lib/Jsrt/Jsrt.cpp

```diff
@@ -1332,6 +1689,23 @@ CHAKRA_API JsGetTypedArrayInfo(_In_ JsValueRef typedArray, _Out_opt_ JsTypedArra
             *byteLength = typedArrayBase->GetByteLength();
         }
     }
+
+#if ENABLE_TTD
```

Comment:
Chakra can create a TypedArray internally and pass it to the host (which is what we have in some Node benchmarks) so we support this even though the explicit JsRT JsCreateTypedArray is not supported yet.